frontend/model/contracts/group.js

```diff
@@ -441,28 +568,51 @@ DefineContract({
           }
         }
       },
-      process ({ data, meta }, { state }) {
-        Vue.delete(state.profiles, data.member)
+      process ({ data, meta }, { state, getters }) {
+        state.profiles[data.member].status = PROFILE_STATUS.REMOVED
+        state.profiles[data.member].departedDate = meta.createdDate
```

Comment:
Why the need to change `removeMember` logic in a PR related with payments stuff?

Reply:
This was done because of the way that the payments distributions were calculated in a previous commit, which relied on keeping track of who was an active member during each month, in order to calculate the distribution for that month.

In a later commit I stopped doing it that way, but did not revert this part. I decided to keep it so that who said what in chat histories could be preserved. Margarida also has designed a profile card for members who are no longer active in the group, and this makes it easier to implement that.